src/Classes/Blockchain.store.ts

```diff
@@ -107,7 +109,7 @@ export class BlockchainStore {
     this._clearPersistentSession()
 
     // Set Blockchain store state
-    this._blockchain.clearWallet()
+    this._blockchain.clearWallet(true)
```

Comment:
why is a boolean necessary? ```clearWallet``` should call ```this._provider.disconnect()``` upon logout only if the current provider is mew wallet anyway, or am i missing something?

Reply:
we need to call this._provider.disconnect()  for mew if you  call it via ofi ui. But there is a disconnect button on mew app, and it calls disconnect automatically, so we don't need to call it againg